install_config/aggregate_logging.adoc

```diff
@@ -12,628 +12,475 @@ toc::[]
--
-  apiVersion: "v1"
-  kind: "Service"
-  metadata:
-    labels:
-      provider: "fabric8"
-      component: "elasticsearch"
-    name: "es-logging"
-  spec:
-    ports:
-    -
-      port: 9200
-      targetPort: 9200
-    selector:
-      provider: "fabric8"
-      component: "elasticsearch"
--
-  apiVersion: "v1"
-  kind: "Service"
-  metadata:
-    labels:
-      provider: "fabric8"
-      component: "elasticsearch"
-    name: "es-logging-cluster"
-  spec:
-    portalIP: "None"
-    ports:
-    -
-      port: 9300
-      targetPort: 9300
-    selector:
-      provider: "fabric8"
-      component: "elasticsearch"
--
-  apiVersion: "v1"
-  kind: "ReplicationController"
-  metadata:
-    labels:
-      provider: "fabric8"
-      component: "elasticsearch"
-    name: "elasticsearch"
-  spec:
-    replicas: 1
-    selector:
-      provider: "fabric8"
-      component: "elasticsearch"
-    template:
-      metadata:
-        labels:
-          provider: "fabric8"
-          component: "elasticsearch"
-      spec:
-        serviceAccount: es-deploy
-        containers:
-          -
-            securityContext:
-              runAsUser: 0
-            env:
-            -
-              name: "KUBERNETES_TRUST_CERT"
-              value: "true"
-            -
-              name: "SERVICE_DNS"
-              value: "es-logging-cluster"
-            image: "fabric8/elasticsearch-k8s:1.5.2"
-            name: "elasticsearch"
-            ports:
-            -
-              containerPort: 9200
-              name: "http"
-            -
-              containerPort: 9300
-              name: "transport"
+$ oc get dc --selector logging-infra=elasticsearch
 ----
 ====
 
-Create the objects using the new file:
+////
+To scale ElasticSearch deployments, create and add more deployments, being aware
+of the cluster parameter restrictions. The deployer uses a template to create
+ElasticSearch deployments. These deployments will be named differently, but will
+all have the 'service/logging-es-cluster' prefix:
 
 ====
 ----
-$ oc create -f path/to/elasticsearch.yaml
+$ oc process logging-es-template | oc create -f -
 ----
 ====
+////
+
+*ElasticSearch Storage*
 
-This starts a single *Elasticsearch* instance. If you need to create a larger
-cluster, you can scale the *Elasticsearch* replication controller using:
+The deployer creates an ephemeral deployment in which all of a pod's data is
+lost upon restart. For production, persistent storage is recommended. The
+following example specifies a persistent storage volume for the ElasticSearch
+deployment. You can use the `oc volume` command to add a created volume to a
+deployment:
 
 ====
 ----
-$ oc scale --replicas=3 rc elasticsearch
+$ oc volume dc/logging-es-rca2m9u8 \
+          --add --overwrite --name=elasticsearch-storage \
+          --type=hostPath --path=/path/to/storage
 ----
 ====
 
-[[creating-logging-pods]]
+You can any volume type, such as
+link:../admin_guide/persistent_storage_nfs.html[NFS].
```

Comment:
This file got moved, link should be `../install_config/persistent_storage/persistent_storage_nfs.html`